javascript/react/redux-tips/async-flow-in-redux.md

```diff
@@ -30,31 +31,65 @@ The default **Redux** store only supports synchronous data flow. However, with t
 
 Packages such as `redux-thunk` and `redux-promise` provide syntax sugar as to help you achieve such data flow.
```

Comment:
Actually this was wrong. 

These libraries do not change the syntax to be prettier or easier to read, they add flow control logic.

```suggestion
Dealing with async operations in Redux is usually achieved with external libraries that enhance Redux with additional capabilities allowing you to handle async behavior.

Common strategies take advantage of promises by allowing them to be used somewhere in the Redux data flow: returned from action dispatching, sent via action creators, etc. With any strategy, an async library will typically send an initial action when an async operation begins (a promise is created), then send another action (either success or error) when the async operation ends (a promise is settled).

Some well-known libraries are `redux-thunk` and `redux-promise`.
```